views/homepage.ejs

```diff
@@ -34,8 +34,8 @@
                             <p class="col-xs-2 col-xs-offset-5">Sign Up</p>
                         </div>
                         <div class="row">
-                            <p class="col-xs-offset-2 col-xs-2">Email</p>
-                            <input ng-model="signup.email" type="text" class="col-xs-4" required>
+                            <p class="col-xs-offset-2 col-xs-2">Username</p>
```

Comment:
Lol thank you for maintaining this but omg this is sooooo old and not used. This whole file is completely deprecated haha oops